src/foam/u2/view/UnstyledTableView.js

```diff
@@ -293,6 +281,7 @@ foam.CLASS({
 
             return this.E().
               addClass(view.myClass('tr')).
+              style({ 'min-width': this.tableWidth_$ }).
```

Comment:
Does tableWidth_ ever change its value?

Reply:
it should be on column selection change as it's the sum of the columns tableWidth